functional_tests/steps/bundles.py

```diff
@@ -67,3 +69,52 @@ def the_selected_datasets_are_displayed(context: Context) -> None:
     context.page.get_by_text(
         "Deaths registered weekly in England and Wales by region (Edition: Example Dataset 3, Ver: 1)"
     ).is_visible()
+
+
+# bundle create amend
+@step('a bundle has been created with name "{bundle_name}" and creator')
```

Comment:
I don't think it's worth parametrising this step tbh, we can just rely on the BundleFactory to generate it. 

Similarly in `bundle_menu_contains_value` and `bundle_menu_does_not_contain_value` you don't need to pass the bundle name as an argument and simply do `bundle.context.name`